pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -123,30 +124,21 @@ def lightning_module(self) -> 'pl.LightningModule':
         """Returns the pure LightningModule without potential wrappers"""
         return unwrap_lightning_module(self._model)
 
-    @property
-    def results(self) -> Any:
```

Comment:
please leave it. read only is better and I can improve the docs. this is only here for handling the ddp-spawn case internally.

Reply:
Done